src/components/select/filterable-select/filterable-select.component.tsx

```diff
@@ -125,6 +128,7 @@ export const FilterableSelect = React.forwardRef(
       inputRef,
       enableVirtualScroll,
       virtualScrollOverscan,
+      disableFiltering = false,
```

Comment:
Can I suggest this prop name is renamed to something like `disableDefaultFiltering`? I was struggling to understand the purpose of this, initially thinking filtering was disabled altogether and therefore what was the purpose of this change to Filterable Select at all if no filtering was then allowed until it was explained to me.